crates/sui-indexer/migrations/mysql/2024-08-14-000000_full_objects_history/up.sql

```diff
@@ -0,0 +1,16 @@
+-- This table will store every history version of each object, and never get pruned.
+-- Since it can grow indefinitely, we keep minimum amount of information in this table for the purpose
+-- of point lookups.
+CREATE TABLE full_objects_history (
+    object_id                   BLOB          NOT NULL,
+    object_version              BIGINT        NOT NULL,
+    object_status               SMALLINT      NOT NULL,
+    serialized_object           MEDIUMBLOB,
+    CONSTRAINT full_objects_history_pk PRIMARY KEY (object_id(32), object_version)
+);
+
+INSERT INTO full_objects_history (object_id, object_version, object_status, serialized_object)
+SELECT object_id, object_version, object_status, serialized_object
+FROM objects_history;
```

Comment:
sorry that I actually missed this, I thought this pr only defined the schema. this will not work as `INSERT INTO` will only run when we set up schema, and `objects_history` is empty then.

I think instead we will want to 
1. remove this `INSERT INTO`
2. add separate ingestion codes to directly write to `full_objects_history` besides `objects_history`

and then when we deploy this pr + the pruning policy pr, we 
1. run both `CREATE TABLE full_objects_history` and `INSERT INTO` via psql
2. update the writer pod with this pr + pruning policy pr and resume writer
3. there will be a small gap after the last cp done by`INSERT INTO` and the first checkpoint done by new ingestion codes, and we can run a `INSERT INTO` again to patch that.